src/app/create-event/page.tsx

```diff
@@ -16,23 +16,20 @@ export default function CreateEvent() {
   const [earliestTime, setEarliestTime] = useState('9:00 AM')
   const [latestTime, setLatestTime] = useState('5:00 PM')
   const [mode, setMode] = useState('weekly')
-  const [config, setConfig] = useState<string[]>([])
+  const [config, setConfig] = useState<string[]>([]) // daysOfWeek
   const [timezone, setTimezone] = useState('')
 
-  const [isAvailable, setIsAvailable] = useState(false) // set to true when name is entered at sign in
+  const [isAvailable, setIsAvailable] = useState(false) // set to true when name is entered at sign in, Determines if the grid is selectable (selection mode)
   const [userName, setUserName] = useState('') // set to name entered at sign in
-  const [responders, setResponders] = useState<string[]>([]) // array to store names of users who signed in
   const dialogRef = useRef<HTMLDialogElement>(null) // modal
 
   const [isButtonsVisible, setIsButtonsVisible] = useState(false) // New state to control visibility of buttons
 
-  /* 
-    Add userName to responders array when they click the "Save" button 
-   */
-
   //added router to redirect to view-event page after creating event
   const router = useRouter()
 
+  // Revisit: append local to remote api call for selecting all responder usernames
```

Comment:
TODO's are more common. There's a few tools like VS Code that can find anything marked TODO

Reply:
got it! Changing this in next commit